src/entities/applet/model/hooks/useStartEntity.ts

```diff
@@ -151,12 +153,23 @@ function useStartEntity({
           onRestart: () => {
             cleanUpMediaFiles({ activityId, appletId, eventId, order: 0 });
 
+            logger.log(
+              `[useStartEntity.startActivity]: Activity "${entityName}|${activityId}" restarted`,
+            );
             activityStarted(appletId, activityId, eventId);
             resolve({ startedFromScratch: true });
           },
-          onResume: () => resolve({ startedFromScratch: false }),
+          onResume: () => {
+            logger.log(
+              `[useStartEntity.startActivity]: Activity "${entityName}|${activityId}" resumed`,
+            );
+            return resolve({ startedFromScratch: false });
+          },
         });
       } else {
+        logger.log(
```

Comment:
Please clarify to Veronika, aren't these new logs sensitive user data like user actions (user events) that collected during activity execution?